rasa/shared/nlu/training_data/training_data.py

```diff
@@ -38,139 +38,37 @@
 TF_RECORD_KEY_SEPARATOR = "#"
 
 
-class TrainingData:
-    """Holds loaded intent and entity training data."""
+class NLUPipelineTrainingData:
```

Comment:
Do we want to add `NLU` to the name? With e2e we add a lot of not strictly NLU examples and I'm sure we'll breach that abstraction even further in future. How about just `PipelineTrainingData`?

Reply:
that's why I added `NLUPipeline` together, because we add e2e data to be processed by NLU pipeline, I don't think it'll be clear what simply `Pipeline` is

Reply:
Okay, let's roll with it then 👍